### DISCUSSION TITLE
RBAC with Azure AD

### DISCUSSION THREAD

**Discussion by rjames4558** [provectus/kafka-ui#3209] [Q&A] [answered]
Does the RBAC feature currently support Azure AD ? I noticed a few OAuth identity providers listed, and since Azure AD also is OAuth compliant thought possibly that may be an option as well.

If Azure AD is currently supported, what is the subject fetch mechanism ?

**Comment by Haarolean** [comment, marked as answer]
Hi, currently we support just github, cognito and google as oauth providers. Though we'll implement a support for generic oauth providers soon, within the current release, you'll be able to use it as openid provider. Or, alternatively, you could use it as LDAP-AD provider, support for which we'll also implement soon.
